coverage/coverage.txt

```diff
@@ -2,9 +2,4 @@
 Incomplete test coverage
 --------------------------------------------------------------------------------
 
-/app/controllers/view_components_controller.rb: 97.22% (missed: 49)
-/lib/view_component/base.rb: 97.7% (missed: 165,244,272,322)
-/lib/view_component/preview.rb: 92.31% (missed: 32,42,78)
-/lib/view_component/render_to_string_monkey_patch.rb: 83.33% (missed: 9)
-/lib/view_component/test_helpers.rb: 95.45% (missed: 17)
-/test/view_component/integration_test.rb: 97.16% (missed: 14,15,16,18,20)
+100% test coverage. You're all set, friend!
```

Comment:
@joelhawksley I think this is connected to the changes I made to the test setup. I would love to get some advice on how to setup the second dummy app or swap the test environment in order to test a different initializing flow on the engine. 
The way I solved it also brings an issue when running tests and passing a test file name `rake test TEST=...` since I hardcoded in one rake task the file that should be executed on the other environment. Left another comment regarding this here: https://github.com/github/view_component/pull/358#discussion_r441476057